app/components/Views/confirmations/components/Confirm/Info/Info.tsx

```diff
@@ -67,9 +70,14 @@ const Info = ({ route }: InfoProps) => {
 
   if (!InfoComponent) return null;
 
-  if (transactionType === TransactionType.stakingUnstake) {
-    const StakingWithdrawalComponent = InfoComponent as React.ComponentType<UnstakeConfirmationViewProps>;
-    return <StakingWithdrawalComponent route={route as UnstakeConfirmationViewProps['route']} />;
+  if (
+    transactionType && [
+      TransactionType.stakingUnstake,
+      TransactionType.stakingClaim,
+    ].includes(transactionType)
+  ) {
```

Comment:
```suggestion
  ) {
  // Unstake and staking claim pages need route parameters to be passed into Info components (`StakingClaim` & `StakingWithdraw`)
```
Shall we add a comment here why we don't have `stakingDeposit` for future references?